.github/workflows/smoketests.yml

```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
     name: Build
-    if: "!github.event.pull_request.head.repo.fork"
```

Comment:
@vmanilo is this correct to merge or something you did to debug on your repo?

Reply:
yes, smoke tests triggers on cron schedule, so it's not a pull request event, that's why we need to fix this statement